### ISSUE TITLE
[Bug] Cannot change the color of waypoints

### ISSUE THREAD

**Issue by Theboiboi8** [MatHax/Client#76]
### Describe the bug

Then you change the color of a waypoints it stays to the default and doesn't change

### Steps to reproduce

```bash
Install MatHax
Enable Waypoints
Add a waypoint
Try Changing the color
Notice how the color reverts to the default rose-red
```

### MatHax Version

1.7.1

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by Matejko06** [comment]
Fixed for 1.7.6

**Comment by Matejko06** [comment]
Fixed for 1.7.6
